### DISCUSSION TITLE
Regression with multiple objective values

### DISCUSSION THREAD

**Discussion by anthonysirico** [pyg-team/pytorch_geometric#7988] [Q&A]
Are there any examples of multiple output regression?

For example, I have a graph dataset, where each graph has multiple graph-level regression values.

**Comment by rusty1s** [comment]
You can take a look at the `qm9_nn_conv` example. If you drop the `target = 0` line and adjust the model output dimension, then this model will be able to regress multiple outputs at once. It is not commonly used on QM9 though since it usually performs worse than learning a single model for each target.

**Reply by anthonysirico** [reply]
I will take a look, thank you.
